docs/websites/nginx/how-to-install-nginx-and-a-startssl-certificate-on-debian-8-jessie.md

```diff
@@ -1,250 +0,0 @@
----
```

Comment:
I am trying to figure out why this deleted file shows up in this PR. I understand that it's an older version of another guide the original author had accepted under a new name. Since that means there's no guide in master by that name, why is it showing as a deletion at all?

In any case, it shouldn't be a problem, but whoever merges this needs to be sure to check this guides' final form, `install-nginx-and-a-startssl-certificate-on-debian-8-jessie.md`